pkg/services/ngalert/models/alert_rule.go

```diff
@@ -79,10 +79,16 @@ const (
 	NamespaceUIDLabel = "__alert_rule_namespace_uid__"
 
 	// Annotations are actually a set of labels, so technically this is the label name of an annotation.
+	StateReasonAnnotation = "StateReason"
+
 	DashboardUIDAnnotation = "__dashboardUid__"
 	PanelIDAnnotation      = "__panelId__"
 )
 
+var (
+	StateReasonMissingSeries = "MissingSeries"
```

Comment:
not sure about this. Suggestions are welcome.
This annotation also can be used in this case https://github.com/grafana/grafana/pull/49259

Reply:
You could invert the words, some options could be:

- `SeriesAbsent`
- `SeriesMissing`